src/tests/gameBoard.test.js

```diff
@@ -0,0 +1,51 @@
+import { Ship } from "../ship";
+import { GameBoard } from "../gameBoard"
+
+let board;
+
+beforeEach(() => {
+    board = GameBoard (); 
+})
+
+it('check whether gameBoard can successfully create ship on X axis', () => {
```

Comment:
It looks like this test (and the next one) mostly test `Ship.hit` and `Ship.isSunk`.